ports/boringssl/portfile.cmake

```
@@ -28,6 +28,7 @@ vcpkg_from_github(
   PATCHES
     0001-vcpkg.patch
     0002-disable-waring-4065.patch
+    0003-Disable-C4191-warning-to-fix-VS2022-build.patch
 )
 
 vcpkg_configure_cmake(
```

```suggestion
vcpkg_cmake_configure(
```
`vcpkg_configure_cmake` has been deprecated, and please also remove `PREFER_NINJA`. Change `vcpkg_install_cmake()` to `vcpkg_cmake_install()`

---

Also add dependency to vcpkg.json
```
"dependencies": [
    {
      "name": "vcpkg-cmake",
      "host": true
    }
]
```